COMPILING.md

```diff
@@ -316,6 +331,21 @@ For SDL:
 
 For `app` builds, launch Cataclysm.app from Finder.
 
+### dmg distribution
+
+You can build a nice dmg distribution file with the `dmgdist` target. You will need a tool called [dmgbuild](https://pypi.python.org/pypi/dmgbuild). To install this tool, you will need Python first. If you are on Mac OS X >= 10.8, Python 2.7 is pre-installed with the OS. If you are on an older version of OS X, you can download Python [on their official website](https://www.python.org/downloads/) or install it with homebrew `brew install python`. Once you have Python, you should be able to install `dmgbuild` by running:
+
+    # This install pip. It might not be required if it is already installed.
+    curl --silent --show-error --retry 5 https://bootstrap.pypa.io/get-pip.py | sudo python
+    # dmgbuild install
+    sudo pip install dmgbuild pyobjc-framework-Quartz
+
+Once `dmgbuild` is installed, you will be able to use the `dmgdist` target like this:
+
+    make dmgdist NATIVE=osx OSX_MIN=10.7 RELEASE=1 TILES=1 FRAMEWORK=1 LOCALIZE=0 CLANG=1
```

Comment:
This example `make` command should include `USE_HOME_DIR=1`. Otherwise the config and save files end up inside the .app and they will be lost when upgrading.